# --- ROCK-COPYRIGHT-NOTE-BEGIN ---
# 
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# Please add additional copyright information _after_ the line containing
# the ROCK-COPYRIGHT-NOTE-END tag. Otherwise it might get removed by
# the ./scripts/Create-CopyPatch script. Do not edit this copyright text!
# 
# ROCK Linux: rock-src/package/daja77/zd1211/zd1211-makefile.patch
# ROCK Linux is Copyright (C) 1998 - 2006 Clifford Wolf
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# 
# --- ROCK-COPYRIGHT-NOTE-END ---

--- ./Makefile.vanilla	2005-12-09 15:52:39.000000000 +0100
+++ ./Makefile	2005-12-09 15:54:43.000000000 +0100
@@ -73,7 +73,7 @@
 SRC_DIR=$(PWD)/src
 
 KVER := $(shell echo $(KERNRELEASE)|cut -d "." -f1-2;)
-MODPATH := /lib/modules/$(KERNRELEASE)
+MODPATH := /lib/modules/$(KERNRELEASE)/kernel/drivers
 
 # Retrieving driver version
 ZD_VERSION := $(shell $(DATE) +%Y%m%d)
@@ -115,7 +115,7 @@
 # for apdbg
 	gcc -o apdbg tools/apdbg.c
 	chmod +x apdbg
-	cp ./apdbg /usr/local/sbin/apdbg   
+	cp ./apdbg /usr/sbin/apdbg   
 
 clean:
 	@$(RM) .tmp_versions .*.cmd *.ko *.mod.c *.mod.o *.o $(SRC_DIR)/*.o  $(SRC_DIR)/.*.o.cmd $(SRC_DIR)/*~ $(SRC_DIR)/zddevlist.h
